src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/DefaultProcessConfigurationTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.RestServer;
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.UnitTests
+{
+    public class DefaultProcessConfigurationTests
+    {
+        [Fact]
+        public void ConvertProcessKeyTest()
+        {
+            var processKey = new ProcessKey("processName");
+            var filter = CreateFilterEntry(processKey);
+            ValidateProcessFilter(filter, DiagProcessFilterCriteria.ProcessName, processKey.ProcessName);
+
+            processKey = new ProcessKey(Guid.NewGuid());
+            filter = CreateFilterEntry(processKey);
+            ValidateProcessFilter(filter, DiagProcessFilterCriteria.RuntimeId, processKey.RuntimeInstanceCookie.Value.ToString("D"));
+
+            processKey = new ProcessKey(5);
+            filter = CreateFilterEntry(processKey);
+            ValidateProcessFilter(filter, DiagProcessFilterCriteria.ProcessId, processKey.ProcessId.Value.ToString(System.Globalization.CultureInfo.InvariantCulture));
```

Comment:
nit: Add System.Globalization as a using to minimize the length of this statement.